Applications/PlazaDetails/PublishService.qmd

```diff
@@ -9,88 +9,82 @@ Users can publish their algorithm as a service in the openEO Algorithm Plaza. Th
-As a user, you can click on the avatar or the service's name to load its detail page. 
-In addition, you can click on the Register service CTA button to load the add service page to create a new service.
+Already created services display an avatar (at this stage, just a colour and initials from the service name), the service name, and a service status indication. Users can click on the avatar or the service's name to load its detail page. Additionally, users can click on the "Register service" CTA button to load the add service page and create a new service.
 
-Furthermore, together with the service cards you have created, you can see a card area with a `REGISTER NEW SERVICE` CTA button. This button will load the add service page to create a new service.
+Furthermore, together with the service cards created, users can see a card area with a `REGISTER NEW SERVICE` CTA button. This button will load the add service page to create a new service.
 
 ## Develop an openEO algorithm
 
-A good integration in openEO algorithm plaza begins as you start programming your algorithm.
+A good integration in openEO algorithm plaza begins as user starts with their algorithm.
 
 In openEO, a 'datacube' view is used, which hides a lot of the complexity when working with huge EO data archives.
-It provides full archive access to the most popular datasets. You will have to get familiar with the Application Programming Interface (API), which provides a lot of predefined processes.
-To integrate existing code, you will have to use the concept of 'User Defined Functions' ([UDFs](https://open-eo.github.io/openeo-python-client/udf.html){target="_blank"}). Furthermore, Parallelization and scalability are taken care of.
+It provides full archive access to the most popular datasets. 
+To integrate existing code, the concept of 'User Defined Functions' ([UDFs](https://open-eo.github.io/openeo-python-client/udf.html){target="_blank"}) can be used. Furthermore, Parallelization and scalability are taken care of.
 
 #### Working with openEO
 
-To get familiar in working we openEO we recommend to start with a basic introduction on using openEO can be found [here](https://documentation.dataspace.copernicus.eu/APIs/openEO/openEO.html){target="_blank"}. Furthermore, to deploy your openEO algorithm as a service, you can use the ['user defined process'](https://open-eo.github.io/openeo-python-client/udp.html){target="_blank"} functionality.
+To get familiar in working we openEO we recommend to start with a basic introduction on using openEO can be found [here](https://documentation.dataspace.copernicus.eu/APIs/openEO/openEO.html){target="_blank"}. Furthermore, to deploy an openEO algorithm as a service, the ['user defined process'](https://open-eo.github.io/openeo-python-client/udp.html){target="_blank"} functionality can be used.
```

Comment:
please reread this sentence carefully. it contains quite a lot of grammatical errors